commitm

```diff
@@ -6,38 +6,40 @@ use strict;
 use FindBin qw($Bin $Script);
 use YAML::XS qw(LoadFile);
 use Getopt::Long;
-use Readonly;
 use Try::Catch;
 
-Readonly::Scalar my $MSG_FILE => "$Bin/$Script.yml";
-
 sub main {
-    my $lang = '*';
-    try {
-        GetOptions('language=s', \$lang);
-    }
-    catch {
-        die 'Failed getting command line arguments.';
+    my $lang = '_';
+
+    my $load_messages = sub {
+        my ($lang) = @_;
+
+        my $msgs;
+        my $msg_file = "$Bin/messages/$lang.yml";
+
+        try {
+            $msgs = LoadFile($msg_file);
+        }
+        catch {
+            die "Failed loading $msg_file -> $_";
+        };
+
+        return $msgs;
     };
 
-    my $msgs;
     try {
-        $msgs = LoadFile($MSG_FILE);
+        GetOptions('language=s', \$lang);
     }
     catch {
-        die "Failed loading $MSG_FILE";
+        die 'Failed getting command line arguments.';
     };
 
-    unless (ref $msgs eq 'HASH') {
-        die "Key ‘$lang’ not found";
-    }
-    unless ($msgs = $msgs->{$lang}) {
-        die "No messages found for ‘$lang’";
-    }
+    my $msgs = $load_messages->($lang);
 
     unless (ref $msgs eq 'ARRAY') {
-        die "No message list for ‘$lang’ found";
+        die "No ‘$lang’ lang messages";
```

Comment:
Sorry, inadvertent change, while I was mucking around with dumper. Your message is perfectly fine.

Reply:
Reverted.